src/bin.js

```diff
@@ -50,13 +49,23 @@ try {
 
 async function getmonitors(auth, monitorid, monitorname) {
   try {
-    const response = await request.get('https://synthetics.newrelic.com/synthetics/api/v3/monitors', {
-      headers: {
-        'X-Api-Key': auth,
-      },
-      json: true,
-    });
-    const monitors = response.monitors.map(({ id, name }) => ({ id, name }));
+    let more = true;
```

Comment:
Please resolve the conflicts with the base version first. The log addition itself looks good.